STACpopulator/extensions/rdps.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+from pydantic import model_validator
+
+from STACpopulator.extensions.cf import CFHelper
+from STACpopulator.extensions.file import FileHelper
+from STACpopulator.extensions.thredds import THREDDSCatalogDataModel
+
+
+# Customize the THREDDSCatalogDataModel
+class RDPSDataModel(THREDDSCatalogDataModel):
+    """Data model for RDPS NetCDF datasets."""
+
+    # Extension classes
+    cf: CFHelper
+    file: FileHelper
+
+    @model_validator(mode="before")
+    @classmethod
+    def cf_helper(cls, data: dict) -> dict:
+        """Instantiate the cf helper."""
+        data["cf"] = CFHelper(variables=data["data"]["variables"])
+        return data
+
+    @model_validator(mode="before")
+    @classmethod
+    def file_helper(cls, data: dict) -> dict:
+        """Instantiate the file helper."""
+        data["file"] = FileHelper(access_urls=data["data"]["access_urls"])
+        return data
```

Comment:
I am aware this is based on the `THREDDSCatalogDataModel`. Asking more broadly. @huard 

Should the helpers be invoked using something like `cls.cf(...)` rather than repeating the class, to make sure we only update the field annotation to apply everywhere?

Not blocker if not.

Reply:
Yes, you're absolutely right.